csrc/preseg_passes/move_pad.cpp

```diff
@@ -287,7 +287,10 @@ TensorView* replayConcretePad(
 
   auto* new_out = IrBuilder::create<TensorView>(
       IrBuilder::create<TensorDomain>(
-          merged_root_ids, merged_logical_ids, merged_logical_ids),
+          merged_root_ids,
+          merged_logical_ids,
+          merged_logical_ids,
+          TensorDomain::getContiguityFilledWith(merged_logical_ids, true)),
```

Comment:
Great question. No I don't need this change to pass the added tests. Since we only check contiguity on IO tensors.

But seems like the right thing to do.